src/raphael_backend_flask/db.py

```diff
@@ -133,9 +148,20 @@ def create_database(path: str) -> None:
     create_table(db, table_claim_extraction_runs)
     create_table(db, table_training_claims)
     create_table(db, table_inferred_claims)
+    create_table(db, table_users)
+
+    default_username = "fullfact"
+    default_password = generate_password_hash("changeme")
+    try:
```

Comment:
It would be better to handle this in SQL as follows:
```
INSERT INTO users (username, password_hash, admin) 
VALUES (?, ?, TRUE)
ON CONFLICT(username) DO NOTHING;
```

as this should allow us to avoid the try/except. 

I would also ask that the string manipulation is replaced with parameters for the sake of good practice (again, not really a security issue here, but I am pedantic about not using string manipulation to add values to queries).

Reply:
I don't see a reason to be pedantic about it